### PR TITLE
Moved image file and corrected patch in scss and app component files

### PR DESCRIPTION
**Pull request by allisonjw**
**What's this PR do?**
- fixes path to images by adding folder into source
**Where should the reviewer start?**
- see that image paths have been corrected and are showing in virtual DOM

### CODE REVIEW

File: src/Components/App/App.js [hunk 1]
```diff
@@ -27,6 +27,7 @@ class App extends Component {
       <main className="App">
         <Switch>
           <Route exact path='/' render={() => <Login userInfo={this.userInfo}/> } />
+          <Route path='/movies' render={() => <Login userInfo={this.userInfo}/> } />
```
Comment (Asilo5) [inline src/Components/App/App.js]: I think here the component should be MoviesContainer rather than Login

**Review by Asilo5 — CHANGES_REQUESTED** [review]
If you don't mind changing up the route to go to MoviesContainer to then merge that would be gucci =)

File: src/Components/App/App.js (hunk 1 above)
Comment (allisonjw) [reply src/Components/App/App.js]: @Asilo5 I fixed the path for the route. But commented out until the information in the MoviesContainer component it correct and the fetch is working.

File: src/Components/App/App.js [hunk 2]
```diff
@@ -27,6 +28,7 @@ class App extends Component {
       <main className="App">
         <Switch>
           <Route exact path='/' render={() => <Login userInfo={this.userInfo}/> } />
+          {/* <Route path='/movies' render={() => <MoviesContainer userInfo={this.userInfo}/> } /> */}
```
Comment (Asilo5) [inline src/Components/App/App.js:31]: Awesome, thanks!

**Review by Asilo5 — APPROVED** [review]

**Pull request merged**